app.cfg.dist

```diff
@@ -12,3 +12,9 @@ authorize_uri = https://sandbox.orcid.org/oauth/authorize
 token_uri = https://sandbox.orcid.org/oauth/token
 client_id = client_id
 client_secret = client_secret
+
+[bus]
```

Comment:
https://github.com/elifesciences/profiles-formula/blob/master/salt/profiles/config/srv-profiles-app.cfg needs updating, using some pillar values; `{{ pillar.elife.env}}` is good for `env`, `name` can be hardcoded, the other two need configuring.
These are the existing topics:
```
        {
            "TopicArn": "arn:aws:sns:us-east-1:512686554592:bus-profiles--ci"
        }, 
        {
            "TopicArn": "arn:aws:sns:us-east-1:512686554592:bus-profiles--continuumtest"
        }, 
        {
            "TopicArn": "arn:aws:sns:us-east-1:512686554592:bus-profiles--end2end"
        }, 
        {
            "TopicArn": "arn:aws:sns:us-east-1:512686554592:bus-profiles--prod"
        }, 
```

Reply:
I have opened PRs for both Profiles-formula and Builder-private including the new values.